geode-docs/managing/security/method_invocation_authorizers.html.md.erb

```diff
@@ -0,0 +1,165 @@
+| `java.lang.Boolean`                                                       | `booleanValue` |
+| `java.lang.Number`                                                        | `byteValue`, `intValue`, `doubleValue`, `floatValue`, `longValue`, `shortValue` |
+| `java.util.Date`                                                          | `after`, `before`, `getTime` |
+| `java.sql.Timestamp`                                                      | `getNanos` |
+| `java.lang.String`                                                        | `chartAt`, `codePointAt`, `codePointBefore`, `codePointCount`, `compareToIgnoreCase`, `concat`, `contains`, `contentEquals`, `endsWith`, `equalsIgnoreCase`, `getBytes`, `hashCode`, `indexOf`, `intern`, `isEmpty`, `lastIndexOf`, `length`, `matches`, `offsetByCodePoints`, `replace`, `replaceAll`, `replaceFirst`, `split`, `startsWith`, `substring`, `toCharArray`, `toLowerCase`, `toUpperCase`, `trim` |
+| `java.util.Map.Entry`, `org.apache.geode.cache.Region.Entry`              | `getKey`, `getValue` |
+| `java.util.Collection`, `java.util.Map`, `org.apache.geode.cache.Region`  | `get`, `entrySet`, `keySet`, `values`, `getEntries`, `getValues`, `containsKey` |  
+
+The authorizer also provides utilities that can be used by custom implementations to determine whether a method is permanently forbidden or, if the method belongs to <%=vars.product_name%>, whether it's considered safe to be used within a query execution.
+ 
+The methods `getClass`, `readObject`, `readResolve`, `readObjectNoData`, `writeObject` and `writeReplace` are permanently forbidden.
+
+The below table shows those methods that belong to <%=vars.product_name%> and are considered safe (for methods on `org.apache.geode.cache.Region`, the authorizer also verifies that the user has the `DATA:READ:RegionName` permission).
+
+| Class                                     | Allowed Methods                                                                 |
+|-------------------------------------------|---------------------------------------------------------------------------------|
+| `org.apache.geode.cache.Region.Entry`     | `getKey`, `getValue`                                                            |
+| `org.apache.geode.cache.Region`           | `get`, `entrySet`, `keySet`, `values`, `getEntries`, `getValues`, `containsKey` |
+
+### <a id="unrestrictedMethodAuthorizer"></a>UnrestrictedMethodAuthorizer
+
+A flexible `MethodInvocationAuthorizer` that allows any method invocation during the query execution as long as the following conditions are met:
+
+- The method is not considered permanently forbidden by the [RestrictedMethodAuthorizer](method_invocation_authorizers.html#restrictedMethodAuthorizer).
+- The method does not belong to <%=vars.product_name%>, or does belong but it's considered safe by the [RestrictedMethodAuthorizer](method_invocation_authorizers.html#restrictedMethodAuthorizer).
```

Comment:
Should be "does belong but is considered safe"

Reply:
👍